pyV2DL3/eventdisplay/fillRESPONSE.py

```diff
@@ -91,21 +88,21 @@ def check_fuzzy_boundary(par, boundary, tolerance):
 
     """
 
-    fuzzy_diff = np.abs(boundary - par) / boundary
-    if fuzzy_diff < tolerance:
-        logging.warning(
-            "Coordinate tolerance is {0:0.3f} and is within {1:0.3f}".format(
-                fuzzy_diff, tolerance
+    if boundary != 0:
```

Comment:
Apologies to be so accurate here: minor improvement would be to have `if boundary == 0.: return False`, which would allow you to continue without intending a block.

Is there a default return value defined in this function?

Reply:
No, the default return value is not defined for this function.  Yes, better to add `if boundary == 0.: return False`, so having this and defining default return value as `False`, would mean one of two mentioned conditions. This should be fine, right?

Reply:
Yes, this sounds right.